### ISSUE TITLE
Quick Tool Tips

### ISSUE THREAD

**Issue by ahudson2014** [OpenRCT2/OpenRCT2#5393]
**OS:** [Windows 10]
**Version:** 0.0.7
**Commit/Build:**

I think it would be a good idea to have instant tool tips or to have the tool tips show up quicker.  When I first started using OpenRCT2 the tool tips were very helpful but I noticed that they took a while to show up.
- [x] Reproducible in RCT2 (vanilla)?
- [x] Multiplayer?

**Steps to reproduce:**
1. Hover any tool
2. Tool tip shows up almost instantly 

**Screenshots / Video:**

![feature request](https://cloud.githubusercontent.com/assets/25679260/25205657/bbc680e0-2531-11e7-9a86-cd60bf1f1000.PNG)

**Comment by Goddesen** [comment]
Though I can't find it now, I remember there being some discussion on the tooltip timing, and how lowering it any would cause tooltips to pop up annoyingly quick for normal gameplay. Maybe there should be something in the options.

**Comment by Broxzier** [comment]
Removing this check makes them appear almost instantly: https://github.com/OpenRCT2/OpenRCT2/blob/develop/src/openrct2/input.c#L1368

I only tested it for a minute, and personally I don't find it annoying.

**Comment by crorabaugh2015** [comment]
Yes, this is where the feature would check for that option, on line 1368 of input.c

I think it would be a good idea as a feature, that way if someone did find it "annoyingly quick" it could be flipped off

**Comment by Gymnasiast** [comment]
We could lower the delay but I'm not going to have an option for this, it's way too trivial.

**Comment by rwjuk** [comment]
Personally I think the delay is fine as-is. It's in line with that in various other programs (I can't think of many that have them appear instantly - I'm sure they're out there, but it's not common).

**Comment by ot5darva** [comment]
Many games display tooltips instantly like in Starcraft 2. RCT3 displays tooltips after hovering for around one second.

**Comment by daihakken** [comment]
-.5s would be better and avoiding the option which may confuse some newcomers of playing OpenRCT2.

**Comment by AaronVanGeffen** [comment]
I don't object to halving the delay. @Gymnasiast, do you still want to tackle this?

**Comment by Broxzier** [comment]
Tooltips appear to have rather inconsistent delays. When moving the mouse cursor on a widget with a tooltip, the tooltip appears immediately, but when you then move the cursor to be on another widget, it takes a while for the tooltip to appear.

**Comment by Gymnasiast** [comment]
@AaronVanGeffen Sure.
